development/index.md

```diff
@@ -38,40 +52,126 @@ There are three types of issue used for planning work:
  - **Task**: a piece of work that isn't necessarily tied to a specific Epic or Story.
    For example, items related to technical debt or house-keeping chores.
 
-A planning meeting is held at the start of the release. In this meeting the team
-discuss the current backlog of stories, tasks and bugs to propose items to be added
-to the release. All items assigned to a milestone should have an assigned owner.
 
-### Retrospective
+Whenever an issue is raised, it will be reviewed by the Product Owner/CTO and added
+to the Product Board for prioritisation and planning. The exception to this are
+tasks/bugs related to work already in progress and that need to be addressed in
+the current milestone. They should be added to the Development Board and current
+milestone directly.
 
-The day after each FlowForge release, always a Friday, a meeting is scheduled
-by the CTO. This meeting includes 3 parts: a retrospective on the last release,
-complete scheduling for the next release, and kick off development for the next
-release.
+### Project Boards
+
+We use three project boards to plan and track our work.
+
+ - [Roadmap Board](https://github.com/orgs/flowforge/projects/5)
+ - [Product Board](https://github.com/orgs/flowforge/projects/3/views/1)
+ - [Development Board](https://github.com/orgs/flowforge/projects/1/views/1)
+
+#### Roadmap Board
+
+This is a high-level view of our product roadmap over the coming releases. It 
```

Comment:
@sammachin @knolleary I noticed these cards weren't epics. Any reason why? It would be nice to click through so to speak.

Reply:
Because the roadmap doesn't nicely map 1:1 to epics, as Nick said on yesterdays call we're using this other style of board so that we can have plain cards rather than issues in repos.

Reply:
Can we go hybrid? Or is GitHub limiting our engineering process? Now there's multiple sources of truth, or that engineers will look at, and one that's maintained by a PM. The consequence is likely that no-one is happy with the result.

Reply:
The Roadmap isn't purely part of the engineering process its the product process which feeds into that, hence having items that arn't engineering based. We tried just using epics as our roadmap and that ends up complicating the epic backlog with a bunch of non-dev stuff.
For some things on the roadmap we're not even at the epic stage yet, for example edge. 
We can look to include a link to a relevent epic on the Development cards where one exists as the cards are just markdown

Reply:
The Roadmap board is more about forward looking and high level planning, the product and development boards are where the work lives and how you can see the state of what is in progress,